### DISCUSSION TITLE
Broker Load方式从亚马逊 S3导入doris，6700w，54字段数据失败

### DISCUSSION THREAD

**Discussion by yueym1** [apache/doris#26644] [A - General / Q&A]
Broker Load 导入亚马逊 S3, 6700w，54字段数据（doris版本1.2）
做了以下尝试，没有改善：
1.  增加broker节点导入数据上限以及并发min_bytes_per_broker_scanner/max_bytes_per_broker_scanner/max_broker_concurrency
2. 增加导入任务的内存：exec_mem_limit
3. 设置并行度：load_parallelism
每次查看导入结果信息中的JobDetails信心，ScannedRows均为41878572条。
！！！跪求大神帮忙看下！！！
以下是broker 信息，一个broker名字对应于3个be节点的3个broker进程。

以下任务结束后show load展示的信息：
JobId	Label	State	Progress	Type	EtlInfo	TaskInfo	ErrorMsg	CreateTime	EtlStartTime	EtlFinishTime	LoadStartTime	LoadFinishTime	URL	JobDetails	TransactionId	ErrorTablets
1266334	test_s36744w	CANCELLED	ETL:N/A; LOAD:N/A	BROKER		cluster:N/A; timeout(s):14400; max_filter_ratio:0.0	type:LOAD_RUN_FAIL; msg:errCode = 2, detailMessage = tablet error: tablet writer write failed, tablet_id=1254344, txn_id=3082158, err=6, errcode=-1, msg:    @     0x55bf942efd24  doris::Status::ConstructErrorStatus()
    @     0x55bf942f01d5  doris::Status::OLAPInternalError()
    @     0x55bf946e849e  doris::FlushToken::submit()
    @     0x55bf947cda76  doris::DeltaWriter::_flush_memtable_async()
    @     0x55bf947d046a  doris::DeltaWriter::write()
    @     0x55bf94cd2ee2  doris::TabletsChannel::add_batch<>()
    @     0x55bf94da1046  doris::LoadChannel::add_batch<>()
    @     0x55bf94da1383  doris::LoadChannelMgr::add_batch<>()
    @     0x55bf94d92480  (unknown)
    @     0x55bf94baf275  doris::PriorityThreadPool::work_thread()
    @     0x55bf9c569930  (unknown)
    @     0x7fe028293ea5  start_thread
    @     0x7fe0285a6b0d  __clone
    @              (nil)  (unknown)
, host: doris-v122-be-0.doris-v122-be.pbear-dev-system	2023-11-09 14:51:23	2023-11-09 14:55:05	2023-11-09 14:55:05	2023-11-09 14:55:05	2023-11-09 14:58:41		{"Unfinished backends":{"8f6ba52bb6484af9-858acb9515618e9c":[]},"ScannedRows":41878572,"TaskNumber":1,"LoadBytes":30894548487,"All backends":{"8f6ba52bb6484af9-858acb9515618e9c":[28271]},"FileNumber":1,"FileSize":321850115}	3082158	{}

**Comment by LemonLiTree** [comment]
show tablet 1254344;

**Reply by yueym1** [reply]
执行show tablet 1254344结果后; 
`DbName	TableName	PartitionName	IndexName	DbId	TableId	PartitionId	IndexId	IsSync	Order	DetailCmd
default_cluster:A_TEST	ods_mysql_yymtest_tb_0106001_tt_yyyy_mm_dd	ods_mysql_yymtest_tb_0106001_tt_yyyy_mm_dd	ods_mysql_yymtest_tb_0106001_tt_yyyy_mm_dd	30015	1234549	1254335	1234550	true	2	SHOW PROC '/dbs/30015/1234549/partitions/1254335/1234550/1254344';`

**Reply by yueym1** [reply]
执行SHOW PROC '/dbs/30015/1234549/partitions/1254335/1234550/1254344'; 
结果：
`ReplicaId	BackendId	Version	LstSuccessVersion	LstFailedVersion	LstFailedTime	SchemaHash	LocalDataSize	RemoteDataSize	RowCount	State	IsBad	VersionCount	PathHash	MetaUrl	CompactionStatus
1254345	10003	1	1	-1		1655138550	0	0	0	NORMAL	false	1	599929556408181597	http://doris-v122-be-0.doris-v122-be.pbear-dev-system:8040/api/meta/header/1254344	http://doris-v122-be-0.doris-v122-be.pbear-dev-system:8040/api/compaction/show?tablet_id=1254344
1254346	28271	1	1	-1		1655138550	0	0	0	NORMAL	false	1	-1888846208606545495	http://doris-v122-be-1.doris-v122-be.pbear-dev-system:8040/api/meta/header/1254344	http://doris-v122-be-1.doris-v122-be.pbear-dev-system:8040/api/compaction/show?tablet_id=1254344
1254347	28272	1	1	-1		1655138550	0	0	0	NORMAL	false	1	-505680580437398529	http://doris-v122-be-2.doris-v122-be.pbear-dev-system:8040/api/meta/header/1254344	http://doris-v122-be-2.doris-v122-be.pbear-dev-system:8040/api/compaction/show?tablet_id=1254344`

**Reply by yueym1** [reply]
执行 SHOW PROC '/dbs/30015/1234549/partitions';
结果：
`PartitionId	PartitionName	VisibleVersion	VisibleVersionTime	State	PartitionKey	Range	DistributionKey	Buckets	ReplicationNum	StorageMedium	CooldownTime	RemoteStoragePolicy	LastConsistencyCheckTime	DataSize	IsInMemory	ReplicaAllocation
1254335	ods_mysql_yymtest_tb_0106001_tt_yyyy_mm_dd	1	2023-11-06 19:57:35	NORMAL			__dt	10	3	HDD	9999-12-31 23:59:59			0.000 	false	tag.location.default: 3`

**Reply by yueym1** [reply]
执行admin diagnose tablet 1254344;查看tablet状态，均正常
`Item	Info	Suggestion
TabletExist	Yes	
TabletId	1254344	
Database	default_cluster:A_TEST: 30015	
Table	ods_mysql_yymtest_tb_0106001_tt_yyyy_mm_dd: 1234549	
Partition	ods_mysql_yymtest_tb_0106001_tt_yyyy_mm_dd: 1254335	
MaterializedIndex	ods_mysql_yymtest_tb_0106001_tt_yyyy_mm_dd: 1234550	
Replicas(ReplicaId -> BackendId)	{"1254347":28272,"1254346":28271,"1254345":10003}	
ReplicasNum	OK	
ReplicaBackendStatus	OK	
ReplicaVersionStatus	OK	
ReplicaStatus	OK	
ReplicaCompactionStatus	OK	`

**Reply by yueym1** [reply]
另外，同样的表可以导入4000w条数据。

**Reply by LemonLiTree** [reply]
is that 1.2.2 version?

**Reply by yueym1** [reply]
yes，v122
